documentation/feed-using-hadoop-pig-oozie.html

```diff
@@ -90,6 +96,13 @@ <h3 id="feeding-preconstructed-operations">Feeding preconstructed operations</h3
 -- Store into Vespa
 STORE data INTO '$ENDPOINT' USING VespaStorage();
 </pre>
+The source, e.g 'feed-dump.json' needs to contain exactly one valid JSON document operation per line, e.g 
+<pre>
+{"put": "id:namespace:music::0","fields": {"title":"foo"}}
+{"put": "id:namespace:music::1","fields": {"title":"foo"}}
+{"update": "id:namespace:music::2","fields": { "title": { "assign":"bar"}}}
+</pre>
+Hence, pretty-printed JSON will not work.
```

Comment:
Actually, there is a way to load multi-line json: https://github.com/vespa-engine/vespa/blob/master/vespa-hadoop/src/test/pig/feed_multiline_operations.pig

Reply:
Great, added example of that as well.